Sources/Rendering/Core/VolumeProperty/index.js

```diff
@@ -203,6 +203,16 @@ function vtkVolumeProperty(publicAPI, model) {
   publicAPI.getInterpolationTypeAsString = () =>
     macro.enumToString(InterpolationType, model.interpolationType);
 
+  publicAPI.getColorMixCode = (apiSpecificPresets) => {
```

Comment:
I don't think we want such type of "API code" function here.
I suggest VolumeProperty only store a preset value.
For custom code (which will be rarely used), I think this can be done with the "shader replacement" mechanisnm. So you could have 3 presets: "ADDITING/ADDITIVE", "COLORIZING" and "CUSTOM". The custom preset simply adds a "comment" that can be replaced using the shader replacement.
WDYT ?